dbms/src/Flash/Executor/toCPUTimeSecond.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <Flash/Executor/toCPUTimeSecond.h>
+#include <common/likely.h>
+
+namespace DB
+{
+/**
+ * cpu time second used by tiflash compute = vcores * second
```

Comment:
The implementation does not use `vcores`?

Reply:
bad comment removed...
New comment added: `// Convert cpu time nanoseconds to cpu time seconds, and round up.`